src/System.IO.FileSystem/tests/System.IO.FileSystem.Tests.csproj

```diff
@@ -195,8 +196,6 @@
   <ItemGroup>
     <EmbeddedResource Include="Resources\$(AssemblyName).rd.xml" />
   </ItemGroup>
-  <ItemGroup>
-    <Folder Include="Matchers\" />
-  </ItemGroup>
+  <ItemGroup />
```

Comment:
Nit, Redundant

Reply:
Egh- clearly I didn't add that explicitly. :)